stubs/requests/requests/help.pyi

```diff
@@ -0,0 +1,37 @@
+from typing_extensions import TypedDict
+
+# TODO: remove if python/typeshed#8484 is merged
+# from .__version__ import __version__ as requests_version
+
```

Comment:
I don't think we need this TODO comment. The import of this variable is only so that the `info()` and `main()` functions can have the necessary information to work properly. This is an implementation detail that could change at any time. End users should access the `__version__` attribute from `requests.__version__` or `requests.__init__`.

```suggestion
```